lib/cisco_node_utils/cmd_ref/vni.yaml

```diff
@@ -0,0 +1,51 @@
+# vni
+---
+all_vnis:
+  config_get: "show vni"
+  config_get_token: '/^(\d+)\s/'
+
+bridge_domain:
```

Comment:
The 7k code still needs to be refactored as part of US52662. I have made a note of these comments. They will apply when we do US52662.

Here are the methods to be reviewed for now.
feature
feature_set
Create
mapped_vlan
mapped_vlan=(vlan)
default_vlan

Look for this condition 
if /N9K/.match(node.product_id)